README.adoc

```diff
@@ -336,39 +338,55 @@ and that the correct number of albums is returned for each artist in the JSON.
 
 The [hotspot=testJsonBAlbumCountForUnknownArtist]`testJsonBAlbumCountForUnknownArtist` test case is similar to [hotspot=testJsonBAlbumCount]`testJsonBAlbumCount`
 but instead checks an artist that does not exist in the JSON and ensures that a
-value of '-1' is returned.
+value of `-1` is returned.
 
 === Processing with JSON-P test
-ConsumingRestTest.java
+ConsumingRestIT.java
 [source, java, linenums, role='code_column hide_tags=comment']
 ----
-include::finish/src/test/java/it/io/openliberty/guides/consumingrest/ConsumingRestTest.java[]
+include::finish/src/test/java/it/io/openliberty/guides/consumingrest/ConsumingRestIT.java[]
 ----
 
 The [hotspot=testJsonPArtistCount]`testJsonPArtistCount` test uses the `total` endpoint which invokes JSON-P. This test
 checks that deserialization with JSON-P was done correctly and that the correct number
 of artists is returned.
 
+=== Rebuild your WAR file
+
+If you satisfy your application, run the Maven `package` goal to rebuild the WAR file in the `target` directory:
```

Comment:
Is this correct? Shouldn't it be something like " If you are satisfied"?

Reply:
Fixed, thanks!